cluster_management/src/main/java/com/pinterest/rocksplicator/ConfigGenerator.java

```diff
@@ -237,4 +239,27 @@ private boolean updateDisabledHosts() {
     disabledHosts = latestDisabledInstances;
     return true;
   }
+
+  /**
+   * filter out resources with "Task" state model (ie. workflows and jobs);
+   * only return db resources from ideal states
+   *
+   * db_resource and wf/job have to be unique, bcz wf/job is treated as "resource" in helix.
+   * uniqueness will be enforced when workflow/job/db_resource is added
+   */
+  public List<String> filterOutNoneDbResources(List<String> resources) {
```

Comment:
can we rename it to filterOutTaskResources() to make it more accurate to what the function does?